### DISCUSSION TITLE
Multiple dependencies for a template

### DISCUSSION THREAD

**Discussion by Mysterious-Dev** [microsoft/TemplateStudio#4669] [Q&A] [answered]
Hi !

To work on a PR for Template Studio, I need to know, do we have the possibility to set multiple dependencies for a single template ?

![image](https://github.com/microsoft/TemplateStudio/assets/40738104/f5b98f39-bcde-4383-8708-8a4ed62fc1e1)

**Comment by mrlacey** [comment]
You should just be able to pipe-delimit multiple values.
There are existing templates that do this for UWP & WPF.

**Reply by Mysterious-Dev** [reply]
@mrlacey Do you have an example of what template accept multiple values for dependencies (because there are many for UWP 😅) ? Or just an example of the syntax ^^.

**Reply by mrlacey** [reply, marked as answer]
Try these:
https://github.com/microsoft/TemplateStudio/blob/6c5d00743a6cc03f13a7cdd62663ad12e0d72383/code/TemplateStudioForUWP/Templates/Serv/IdentityForcedLogin.CB._VB/.template.config/template.json#L23
https://github.com/microsoft/TemplateStudio/blob/6c5d00743a6cc03f13a7cdd62663ad12e0d72383/code/TemplateStudioForUWP/Templates/Serv/SqlServerDataService._VB/.template.config/template.json#L22
or https://github.com/microsoft/TemplateStudio/blob/6c5d00743a6cc03f13a7cdd62663ad12e0d72383/code/TemplateStudioForWPF/Templates/Pg/XamlIsland/.template.config/template.json#L27


All found with: https://github.com/search?q=repo%3Amicrosoft%2FTemplateStudio+ts.dependencies+language%3AJSON&type=code&l=JSON

**Reply by Mysterious-Dev** [reply]
Lovely ^^ ! Thanks
